spec/routes.spec.ts

```diff
@@ -31,459 +31,474 @@ const testDocConfig = {
   servers: [{ url: 'v1' }],
 };
 
-describe('Routes', () => {
-  describe('response definitions', () => {
-    it('can set description', () => {
-      const registry = new OpenAPIRegistry();
-
-      registry.registerPath({
-        method: 'get',
-        path: '/',
-        responses: {
-          200: {
-            description: 'Simple response',
-            content: {
-              'application/json': {
-                schema: z.string(),
+describe.each`
```

Comment:
No test changes here. Just running the same tests we run against routes against webhooks